docker-compose.dev.local.yml

```diff
@@ -1,3 +1,6 @@
+# This file is generated by `iroha_swarm -p 4 -s Iroha --disclaimer-header --force --config-dir ./configs/peer --build . --outfile docker-compose.dev.local.yml`.
```

Comment:
It seems that this banner tries to put all the parameters so it is self-contained and reproducible only using them. However, the output of `iroha_swarm` also depends on CWD.

I think it is not necessary to put ALL of the parameters here. Most of them you can infer on your own. The only parameter that is impossible (or hard?) to infer is `--seed`.

Reply:
I think it's obvious that this command was executed from the iroha main directory. I'd like to keep all parameters listed because it's hard to infer them. I either have to:

1. Read carefully the whole file, count peers, check if they use image or local build and etc. This is not comfortable and I have to remember all existing parameters, which might be even harder in future if we'll have more of them.
2. Read message from the CI, which I'll now very lately

Reply:
> I think it's obvious that this command was executed from the iroha main directory

It is the case for _our_ docker-compose files, but it is not a general purpose of `iroha_swarm`. This tool is standalone and might be executed from anywhere.

Indeed, the parameters are not easy to infer. Still, I am personally lean against putting all of the parameters into the banner, because CWD **must** be included for completeness, and it might look _dirty_. IMO, the minimal purpose of the banner is only to notice that the file is generated and not hand-written, therefore the reader should probably do some research on their own before editing it by hand.

I wouldn't say I insist on removing the parameters at all, but if we put parameters, CWD should be considered too.

Reply:
Including CWD will lead to reproducibility troubles, especially with how we check in CI that the `yaml` file is up-to-date

Reply:
> Including CWD will lead to reproducibility troubles, especially with how we check in CI that the yaml file is up-to-date

Actually not since I've already modified how we check them in CI -- it skips first 2 lines now.

But looks like I have to rollback it, fine